ui/src/shared/components/dropdown_time_shift/TimeShiftDropdown.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {SFC} from 'react'
+
+import Dropdown from '../../../reusable_ui/components/dropdowns/Dropdown'
```

Comment:
🙅‍♂️ on these relative paths.

Reply:
Blarghhh my text editor did a funky last night and replaced all paths with relative ones in the entire code base, I missed on apparently